lib/compat/wordpress-5.9/class-wp-theme-json-5-9.php

```diff
@@ -14,7 +14,7 @@
  *
  * @access private
  */
-class WP_Theme_JSON_5_9 {
+class WP_Theme_JSON_5_9 extends WP_Theme_JSON {
```

Comment:
This is necessary as the merge function inherits some functions from the original base class.